app/models/reason.rb

```diff
@@ -0,0 +1,3 @@
+class Reason < ActiveRecord::Base
```

Comment:
OK, let's catch up about this tomorrow, ideally with @colinbruce as well. 
Future proofing shouldn't be our concern at the moment and right now this will make things more complicated. For example now can create multiple reasons for 1 application, which is a case we'll have to handle as well. Also the user will have to edit the existing reason, which means the form has to pull data from 2 tables.